### PR TITLE
[task2] - add zombie

### PR DESCRIPTION
**Pull request by varantsou**

### CODE REVIEW

File: Task2/js/app.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+
     const zombieList = document.querySelector('.zombies');
 
-    if (Array.isArray(zombies)){
-        for (let i = 0; i < zombies.length; i++) {
-            renderZombie(zombies[i], i);
+    renderZombieList(zombies);
+
+    function renderZombieList(units) {
+        zombieList.innerHTML = '';
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: well ... it's not a good idea to remove the all zombies on every zombie's dead

File: Task2/js/app.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
 document.addEventListener('DOMContentLoaded', () => {
+    ZOMBIE_TYPE.DEAD = 0;
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: you don't need it, really

**Review by ExSamaelSolbeg — CHANGES_REQUESTED** [review]
[[SCOPE]] implementation is correct

File: Task2/js/app.js [hunk 3]
```diff
@@ -1,37 +1,22 @@
...
+        } else if (zombies[i].type === ZOMBIE_TYPE.MAD) {
             zombie.classList.add('zombie--mad');
-        } else if (item.type === ZOMBIE_TYPE.STRONG) {
+        } else if (zombies[i].type === ZOMBIE_TYPE.STRONG) {
             zombie.classList.add('zombie--strong');
-        } else if (item.type === ZOMBIE_TYPE.DEAD) {
-            zombie.classList.add('zombie--dead');
         }
 
         zombie.addEventListener('click', function (event) {
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: please implement a solution without using any parameter of this function

Comment (varantsou) [reply Task2/js/app.js]: done

File: Task2/js/app.js (hunk 1 above)
Comment (varantsou) [reply Task2/js/app.js]: done

Comment (varantsou) [reply Task2/js/app.js]: done

File: Task2/js/app.js (hunk 2 above)
Comment (varantsou) [reply Task2/js/app.js]: done

File: Task2/js/app.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        if (zombies[i].type === ZOMBIE_TYPE.SMALL) {
+            zombie.classList.add('zombie--small');
+        } else if (zombies[i].type === ZOMBIE_TYPE.MAD) {
+            zombie.classList.add('zombie--mad');
+        } else if (zombies[i].type === ZOMBIE_TYPE.STRONG) {
+            zombie.classList.add('zombie--strong');
+        }
+
+        zombie.addEventListener('click', function () {
+            if (!zombie.classList.contains('zombie--dead')) {
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js:18]: nope
you can't implement this solution without understanding [[ENVIRONMENT]]

**Review by ExSamaelSolbeg — APPROVED** [review]

**Pull request merged**